be/src/exprs/agg/aggregate.h

```diff
@@ -282,7 +282,7 @@ class AggregateFunctionStateHelper : public AggregateFunction {
 public:
     static constexpr bool pod_state() { return std::is_trivially_destructible_v<State>; }
 
-    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const final { new (ptr) State; }
+    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const override { new (ptr) State; }
```

Comment:
Just change here is not enough, consider the nullable aggregate

Reply:
updated, introduce init() after create().